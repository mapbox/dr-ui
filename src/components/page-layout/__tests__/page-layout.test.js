import renderer from 'react-test-renderer';
import { testCases } from './page-layout-test-cases.js';

describe('page-layout', () => {
  describe(testCases.page.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.page;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });

  describe(testCases.example.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.example;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });

  describe(testCases.full.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.full;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });

  describe(testCases.examplePage.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.examplePage;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
