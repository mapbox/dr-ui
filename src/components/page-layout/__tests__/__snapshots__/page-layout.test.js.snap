// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page-layout Basic renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm  col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled none block-mm"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            Some title
          </div>
          <div>
            Some content
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm  col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div>
      Doc content
    </div>
  </div>
</div>
`;

exports[`page-layout Code highlighting test renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm  col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled none block-mm"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            Some title
          </div>
          <div>
            Some content
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm  col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div
      className="prose"
    >
      <h2>
        Has scrollbar
      </h2>
      <pre
        className=" language-curl"
      >
        <code
          className=" language-curl"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </code>
      </pre>
      <h2>
        No scrollbar
      </h2>
      <pre>
        <code>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
        </code>
      </pre>
    </div>
  </div>
</div>
`;

exports[`page-layout Common use case renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm  col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled block"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            <div
              className="ml36"
            >
              Section title
            </div>
          </div>
          <div>
            <div
              className="block-mm none"
            >
              <div
                className="pl24-mm px0 block-mm none pr12 bg-lighten75"
              >
                <div
                  className="py3 "
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-one"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child color-black"
                    >
                      Title one
                      
                    </div>
                  </a>
                  <div
                    className="ml24 pt0"
                  >
                    <ul
                      className="txt-m pb12 inline-block-mm none unprose"
                    >
                      <li
                        className="mb6"
                      >
                        <a
                          className="color-blue-on-hover"
                          href="#heading-one"
                        >
                          Heading one
                        </a>
                        <ul
                          className="none"
                        />
                      </li>
                      <li
                        className="mb6"
                      >
                        <a
                          className="color-blue-on-hover"
                          href="#heading-two"
                        >
                          Heading two
                        </a>
                        <ul
                          className="none"
                        />
                      </li>
                    </ul>
                  </div>
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-two"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title two
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
            </div>
            <div
              className="none-mm block bg-gray-faint px24 py24"
            >
              <div
                className="relative "
              >
                <div
                  className="select-container "
                >
                  <select
                    aria-required={true}
                    className="select select--stroke round-full bg-white"
                    data-test="navigate-this-section-select"
                    disabled={false}
                    id="navigate-this-section"
                    onChange={[Function]}
                    value="page-one"
                  >
                    <option
                      value="page-one"
                    >
                      Title one
                    </option>
                    <option
                      value="page-two"
                    >
                      Title two
                    </option>
                  </select>
                  <div
                    className="select-arrow"
                  />
                </div>
                <div
                  role="alert"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm  col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div>
      <p
        className="mb24"
      >
        Vestibulum id ligula porta felis euismod semper. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum id ligula porta felis euismod semper.
      </p>
      <p
        className="mb24"
      >
        Aenean lacinia bibendum nulla sed consectetur. Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.
      </p>
      <p
        className="mb24"
      >
        Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Curabitur blandit tempus porttitor. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nullam quis risus eget urna mollis ornare vel eu leo. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.
      </p>
      <p
        className="mb24"
      >
        Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Donec sed odio dui. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Integer posuere erat a ante venenatis dapibus posuere velit aliquet.
      </p>
    </div>
  </div>
</div>
`;

exports[`page-layout Custom sidebar column size renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm col--3-ml col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled none block-mm"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            Some title
          </div>
          <div>
            Some content
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm col--9-ml col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div>
      Doc content
    </div>
  </div>
</div>
`;

exports[`page-layout Custom sidebar column size, too large - default to original size renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm  col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled none block-mm"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            Some title
          </div>
          <div>
            Some content
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm  col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div>
      Doc content
    </div>
  </div>
</div>
`;

exports[`page-layout Custom sidebar column size, too small - default to original size renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm  col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled none block-mm"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            Some title
          </div>
          <div>
            Some content
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm  col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div>
      Doc content
    </div>
  </div>
</div>
`;

exports[`page-layout Many first level items renders as expected 1`] = `
<div
  className="grid"
>
  <div
    className="col col--4-mm  col--12 bg-gray-faint"
    data-swiftype-index="false"
  >
    <div
      className="sticky-outer-wrapper"
      style={Object {}}
    >
      <div
        className="sticky-inner-wrapper"
        style={
          Object {
            "position": "relative",
            "top": "0px",
            "zIndex": 3,
          }
        }
      >
        <div
          className="pt24-mm pt0 viewport-almost-mm scroll-auto-mm scroll-styled block"
        >
          <div
            className="txt-l color-blue txt-fancy mb12 block-mm none"
          >
            <div
              className="ml36"
            >
              Section title
            </div>
          </div>
          <div>
            <div
              className="block-mm none"
            >
              <div
                className="pl24-mm px0 block-mm none pr12 bg-lighten75"
              >
                <div
                  className="py3 "
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-one"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child color-black"
                    >
                      Title one
                      
                    </div>
                  </a>
                  <div
                    className="ml24 pt0"
                  >
                    <ul
                      className="txt-m pb12 inline-block-mm none unprose"
                    >
                      <li
                        className="mb6"
                      >
                        <a
                          className="color-blue-on-hover"
                          href="#heading-one"
                        >
                          Heading one
                        </a>
                        <ul
                          className="none"
                        />
                      </li>
                      <li
                        className="mb6"
                      >
                        <a
                          className="color-blue-on-hover"
                          href="#heading-two"
                        >
                          Heading two
                        </a>
                        <ul
                          className="none"
                        />
                      </li>
                    </ul>
                  </div>
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-two"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title two
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-three"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title three
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-four"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title four
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-five"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title five
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-six"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title six
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-seven"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title seven
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-eight"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title eight
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-nine"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title nine
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
              <div
                className="pl24-mm px0 block-mm none pr12"
              >
                <div
                  className="py3  border-t border--gray-light"
                >
                  <a
                    className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
                    href="page-ten"
                  >
                    <div
                      className="pl12 py12 txt-bold txt-m flex-child"
                    >
                      Title ten
                      
                    </div>
                    <div
                      className="flex-child flex-child--no-shrink"
                    >
                      <svg
                        className="events-none icon"
                        focusable="false"
                        role="presentation"
                        style={
                          Object {
                            "height": 18,
                            "width": 18,
                          }
                        }
                      >
                        <use
                          xlinkHref="#icon-chevron-down"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        />
                      </svg>
                    </div>
                  </a>
                  
                </div>
              </div>
            </div>
            <div
              className="none-mm block bg-gray-faint px24 py24"
            >
              <div
                className="relative "
              >
                <div
                  className="select-container "
                >
                  <select
                    aria-required={true}
                    className="select select--stroke round-full bg-white"
                    data-test="navigate-this-section-select"
                    disabled={false}
                    id="navigate-this-section"
                    onChange={[Function]}
                    value="page-one"
                  >
                    <option
                      value="page-one"
                    >
                      Title one
                    </option>
                    <option
                      value="page-two"
                    >
                      Title two
                    </option>
                    <option
                      value="page-three"
                    >
                      Title three
                    </option>
                    <option
                      value="page-four"
                    >
                      Title four
                    </option>
                    <option
                      value="page-five"
                    >
                      Title five
                    </option>
                    <option
                      value="page-six"
                    >
                      Title six
                    </option>
                    <option
                      value="page-seven"
                    >
                      Title seven
                    </option>
                    <option
                      value="page-eight"
                    >
                      Title eight
                    </option>
                    <option
                      value="page-nine"
                    >
                      Title nine
                    </option>
                    <option
                      value="page-ten"
                    >
                      Title ten
                    </option>
                  </select>
                  <div
                    className="select-arrow"
                  />
                </div>
                <div
                  role="alert"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="col col--8-mm  col--12 mt24-mm mb60 pr0-mm px36-mm"
    id="docs-content"
  >
    <div>
      <p
        className="mb24"
      >
        Vestibulum id ligula porta felis euismod semper. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum id ligula porta felis euismod semper.
      </p>
      <p
        className="mb24"
      >
        Aenean lacinia bibendum nulla sed consectetur. Maecenas sed diam eget risus varius blandit sit amet non magna. Vestibulum id ligula porta felis euismod semper. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.
      </p>
      <p
        className="mb24"
      >
        Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Curabitur blandit tempus porttitor. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nullam quis risus eget urna mollis ornare vel eu leo. Praesent commodo cursus magna, vel scelerisque nisl consectetur et. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.
      </p>
      <p
        className="mb24"
      >
        Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Donec sed odio dui. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Integer posuere erat a ante venenatis dapibus posuere velit aliquet.
      </p>
    </div>
  </div>
</div>
`;
